### PR TITLE
Cleanup + 3d model bin file

### PR DESCRIPTION
**Pull request by antworteffekt**

### CODE REVIEW

File: bin/3dmodel [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+
+
+Note: the current implementation uses as estimator a regularized linear model
+with stochastic  gradient descent (SGD) learning scheme: the gradient of the
+loss is estimated each sample at  a time and the model is updated along the way
+with a decreasing strength schedule (aka learning rate). """
+
+epilog = """
+Author: Fabrizio Costa
```
Comment (bgruening) [inline bin/3dmodel]: Not sure about the police here, but you should add your name here as well.

File: eden/converter/molecule/obabel.py [hunk 2]
```diff
@@ -21,28 +17,22 @@ def obabel_to_eden(input, format = 'sdf', **options):
...
         for x in read(input):
             mol = pybel.readstring("sdf", x)
-            #remove hydrogens
+            # remove hydrogens
             mol.removeh()
-            G = obabel_to_networkx(mol)
-            if len(G):
-                yield G
+            g = obabel_to_networkx(mol)
+            if len(g):
```
Comment (bgruening) [inline eden/converter/molecule/obabel.py]: Do you need to check the len()?

File: eden/converter/molecule/obabel.py [hunk 3]
```diff
@@ -21,28 +17,22 @@ def obabel_to_eden(input, format = 'sdf', **options):
...
+            # already converted
             if x not in cache:
                 # convert from SMILES to SDF and store in cache
-                # TODO: do we assume that the input is "clean", i.e. only the SMILES strings?
-                # command_string = 'obabel -:"' + x.split()[1] + '" -osdf --gen3d'
-                # TODO: conformer generation still isn't working - is it a problem in my installation?
-                # command_string = 'obabel -:"' + x + '" -osdf --gen3d --conformer --nconf 5 --score rmsd'
+                # TODO: do we assume that the input is "clean", i.e.
+                # only the SMILES strings?
                 command_string = 'obabel -:"' + x + '" -osdf --gen3d'
```
Comment (bgruening) [inline eden/converter/molecule/obabel.py]: Using pybel or openbabel-python would be cleaner I guess.

**Comment by bgruening** [comment]
I like it, but Travis is still complaining, can you please fix the errors? Thanks!

File: bin/3dmodel (hunk 1 above)
Comment (fabriziocosta) [inline bin/3dmodel]: Sure,do  add your name and email Jose'

File: eden/converter/molecule/obabel.py (hunk 2 above)
Comment (fabriziocosta) [inline eden/converter/molecule/obabel.py]: also, replace g with full variable names, i.e. 'graph'

File: eden/converter/molecule/obabel.py (hunk 3 above)
Comment (fabriziocosta) [inline eden/converter/molecule/obabel.py]: agreed, please look into that Jose'

**Comment by fabriziocosta** [comment]
it seems there is a lot of non cleaned code in the 'old' files. in fact I think we do not need them. wouldn't it be possible to keep them on your file system but remove them from the repository?

**Comment by bgruening** [comment]
You can put file into your `.gitignore` list, these files will not be commited, tracked by git anymore.

**Comment by antworteffekt** [comment]
Right, I hadn't noticed those old files were causing all the errors - I've removed them now.
Regarding using the python openbabel module for conformers, I had thought of doing it that way at first but didn't find a direct way...I'll keep looking into it

**Pull request merged**